### ISSUE TITLE
Android 11: Music Files in folders other than /Music are not visible in Vanilla Music library view 

### ISSUE THREAD

**Issue by cb2-dd** [vanilla-music/vanilla#1096]
- Vanilla Music Version: 1.1.0
- Android Version: 11 (Lineage OS 18.1)
- Phone vendor / model: Motorola Moto G7 Plus
- Application was downloaded from (use `[x]`):
  - [ ] Play Store
  - [x] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [ ] Yes
  - [ ] No
  - [x] NA / Don't know.

## Description
Music files on an external SD card in the folder /Music are addable to the current list of songs to be played in Vanilla Music. But the thousands of cover.jpg files are in the Gallery of the phone > unfortunately, Android 11 does not allow to place a .nomedia file in the folder /Music anymore.

It is possible to create other folders, let's say /Musik and you can place a .nomedia file in there and the problem with the thousands of cover.jpg files is solved. Nevertheless, Vanilla Music is not able to add files via library view from the folder /Musik. All subfolders of /Musik appear empty, no tracks to row in the current playlist. But, from within the Android file manager, it is possible to see those music files and to start their playback in Vanilla Music. But only from the File Manager, not from within Vanilla Music. 

This seems to be a bug effected to Android 11 only?
What would be a solution

**Comment by adrian-bl** [comment]
> Nevertheless, Vanilla Music is not able to add files via library view from the folder /Musik

Did you add the folder to the list of scanned paths?

If not then it's not a surprise that Vanilla won't index your non-standard folder.

**Comment by cb2-dd** [comment]
Yes, I added /Musik to the directory list that is indexed. Even if I only let Vanilla Music index this directory, it does not find something... but from the Android file manager it is possible to play individual files with Vanilla Music...

**Comment by adrian-bl** [comment]
Did you try to tick the 'full filesystem scan' option before starting a manual rescan of the library?

Are the songs (or any folders?) visible in the Files tab of the library?

**Comment by cb2-dd** [comment]
First of all, @adrian-bl, thank you for continuing the discussion.
Yes, i ticket this as well, but for files in SD-card/Musik it does not work... nothing is detected and the Vanilla library files tab shows empty folders. If I use the Android File Manager to copy the existing files from there to SD-card/Music, Vanilla is detecting those files...
(I do not get screenshots uploaded here in Github, otherwise I would show you...)

**Comment by ChiefMikeK** [comment]
Maybe you should post a question/ask4help on the linearge subreddit ?

- https://www.reddit.com/r/LineageOS/

**Comment by cb2-dd** [comment]
Ok, but it looks very confusing there...

Other music programs (e.g. VLC) do not have a problem with files in the folder /Musik so an easier way could be to let Vanilla developers look in the source code of those programs and check out, how they can scan such folders and play the music files in there... from these free music players Vanilla developers could then copy the source code to vanilla music and the problem might be solved... The music player Odyssey can only „see“ file in /Musik if there is no .nomedia file in there. But even without this file (which prevents a cluttered Gallery due to the thousands of cover-images) Vanilla Music is not able to see content of music folders...
